### ISSUE TITLE
Copy event

### ISSUE THREAD

**Issue by nilskp** [KinToday/kin-web-client#41]
Google has a "Copy to my calendar" link for events not in the main calendar. It would be nice to have this, or better, a general "Copy" button to allow copying an event into any writable calendar.

Use case: Some calendars are shared, but others are not. If I attend e.g. a Meetup event, it'd be nice to have that show up in a shared calendar.

**Comment by yperson** [comment]
In a sense, kind of close to [this issue](https://github.com/KinToday/kin-web-client/issues/33).

**Comment by nilskp** [comment]
Yes, so perhaps a copy option for read-only calendars, and copy or move for writable calendars.
